src/drivers/irlock/irlock.cpp

```diff
@@ -0,0 +1,410 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2013 PX4 Development Team. All rights reserved.
```

Comment:
Was this really written 2013? For authors its preferred to use the ```@author``` tag in the file comment below.